client/lib/sites-list/list.js

```diff
@@ -38,9 +38,6 @@ function SitesList() {
 	this.selected = null;
 	this.lastSelected = null;
 	this.propagateChange = this.propagateChange.bind( this );
-
-	PreferencesActions.fetch();
-	this.recentlySelected = PreferencesStore.get( 'recentSites' ) || [];
```

Comment:
I'm getting a number of errors that I think have to do with the removal of these lines when SitesList is constructed. Try going to the stats page, then clearing localStorage, and then reloading.